dev-client/src/screens/CreateSiteScreen/components/CreateSiteForm.tsx

```diff
@@ -90,40 +72,48 @@ export const CreateSiteForm = ({
       <ScrollView>
         <VStack p="16px" pt="30px" space="18px">
           <FormField name="name">
-            <FormLabel>{t('site.create.name_label')}</FormLabel>
             <FormInput placeholder={t('site.create.name_placeholder')} />
           </FormField>
-          <FormField name="coords">
-            <FormLabel>{t('site.create.location_label')}</FormLabel>
-            <Text>
-              {t('site.create.location_accuracy', {
-                accuracyM: accuracyM?.toFixed(0),
-              })}
-            </Text>
-            <FormInput keyboardType="decimal-pad" />
-            <FormControl.Label variant="subtle">
-              {t('site.create.coords_label')}
-            </FormControl.Label>
+
+          <FormLabel>{t('site.create.location_label')}</FormLabel>
+          <Text>
+            {t('site.create.location_accuracy', {
+              accuracyM: accuracyM?.toFixed(0),
+            })}
+          </Text>
+          <FormField name="latitude">
+            <FormInput
+              value={values.latitude.toString()}
+              keyboardType="decimal-pad"
+            />
           </FormField>
-          <FormField name="projectId">
-            <FormLabel>
-              {t('site.create.add_to_project_label')}
-              <HelpTooltipButton>
-                <Text color="primary.contrast" variant="body1">
-                  {t('site.create.add_to_project_tooltip')}
-                </Text>
-              </HelpTooltipButton>
-            </FormLabel>
-            <ProjectSelect
-              projectId={values.projectId ?? null}
-              setProjectId={projectId =>
-                setValues(current => ({
-                  ...current,
-                  projectId: projectId ?? undefined,
-                }))
-              }
+          <FormField name="longitude">
+            <FormInput
+              value={values.longitude.toString()}
+              keyboardType="decimal-pad"
```

Comment:
Issue: Can we keep some label text saying which is latitude vs. longitude?

Reply:
Yes — that will get addressed with the new text field component.

Reply:
Are we guaranteed to implement the new text field component within Capri? Just want to be conscious that we're not regressing functionality in expectation of something that we won't end up shipping.

Reply:
@knipec Yes. It's in review now.